ST_grid.c

```diff
@@ -706,2357 +390,1206 @@ static void _init_grid_files(void)
+		    	gridCells[i][j]._Temp->s = (struct accumulators_st *)
+		               		Mem_Calloc( SuperGlobals.runModelYears,
+ 		                          		sizeof(struct accumulators_st),
+ 		                         		"_allocate_accumulators(Temp)");
+		  	}
+		  	if (BmassFlags.grpb) {
+ 			   	gridCells[i][j]._Grp = (struct stat_st *)
+           				Mem_Calloc( Globals->grpCount,
+                		       sizeof(struct stat_st),
+                		      "_allocate_accumulators(Grp)");
+    			ForEachGroup(rg){
+      				gridCells[i][j]._Grp[rg].s = (struct accumulators_st *)
+             			Mem_Calloc( SuperGlobals.runModelYears,
+                         			sizeof(struct accumulators_st),
+                        			"_allocate_accumulators(Grp[rg].s)");
+				}
 
-}
+    			if (BmassFlags.size) {
+      				gridCells[i][j]._Gsize = (struct stat_st *)
+             				Mem_Calloc( Globals->grpCount,
+                         				sizeof(struct stat_st),
+                        				"_allocate_accumulators(GSize)");
+      				ForEachGroup(rg){
+          				gridCells[i][j]._Gsize[rg].s = (struct accumulators_st *)
+             							Mem_Calloc( SuperGlobals.runModelYears,
+                         							sizeof(struct accumulators_st),
+                        							"_allocate_accumulators(GSize[rg].s)");
+					}
+    			}
+    			if (BmassFlags.pr) {
+      				gridCells[i][j]._Gpr = (struct stat_st *)
+             				Mem_Calloc( Globals->grpCount,
+                		         		sizeof(struct stat_st),
+                		        		"_allocate_accumulators(Gpr)");
+      				ForEachGroup(rg){
+          				gridCells[i][j]._Gpr[rg].s = (struct accumulators_st *)
+             							Mem_Calloc( SuperGlobals.runModelYears,
+                    		     					sizeof(struct accumulators_st),
+                        							"_allocate_accumulators(Gpr[rg].s)");
+					}
+    			}
+
+    			if (BmassFlags.wildfire || BmassFlags.prescribedfire) {
+      				gridCells[i][j]._Gwf = (struct fire_st *)
+             				Mem_Calloc( 1, sizeof(struct fire_st),
+                		        		"_allocate_accumulators(Gwf)");
+
+      				gridCells[i][j]._Gwf->wildfire = (int *) Mem_Calloc( 1,
+                    		  		 sizeof(int) * SuperGlobals.runModelYears,
+                    		  		 "_allocate_accumulators(Gwf->wildfire)");
+      
+      				gridCells[i][j]._Gwf->prescribedFire = (int **) Mem_Calloc( 1,
+                      						sizeof(int **) * SuperGlobals.max_rgroups,
+                       						"_allocate_accumulators(Gwf->prescribedfire");
+
+      				ForEachGroup(rg){
+        				gridCells[i][j]._Gwf->prescribedFire[rg] = (int *)
+          										   Mem_Calloc( SuperGlobals.runModelYears,
+                      										   sizeof(int) * SuperGlobals.runModelYears,
+                      										   "_allocate_accumulators(Gwf->prescribedFire)");
+      				}
+    			}
+  			}
```

Comment:
@kpalmqui this flag is correctly placed. BmassFlags.grpb determines whether or not group statistics should be printed. Since prescribed fire is printed by group it only needs to be allocated if BmassFlags.grpb is on. 

MortFlags.group is roughly the equivalent flag for the mortality output file.

Reply:
The reason the spacing is strange is a mixing of spaces and tabs, which should be fixed when running clang formatter on the project.

Reply:
@chaukap ok thanks!